src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/PerfTests/DebuggerSteppingTestcase.xml

```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <TaoTest xmlns="http://microsoft.com/schemas/VSLanguages/TAO">
   <InitTest>
-    <StartTarget />
+    <StartTarget DontResetOptions="true" />
```

Comment:
resetting option cause all dlls to be loaded which makes checking for loaded dll (CSharp solution only loads CSharp dlls, VB solution only loads VB dlls) fail.

in perf test, we have a script that reset options before running each test separately.